code/modules/cm_tech/droppod/supply.dm

```diff
@@ -7,6 +7,8 @@
 /obj/structure/droppod/supply/open()
 	. = ..()
 	for(var/atom/movable/content as anything in contents)
+		if(istype(content, /obj/structure/closet/crate))
```

Comment:
this feels like a bandaid; how does it get anchored in the first place?

Reply:
(discussed on discord, the crate is anchored to prevent it being shoved pre-transport, so doing this seems alright)